detox/src/ios/earlgreyapi/GREYActions.js

```diff
@@ -4,92 +4,95 @@
 	For more information see generation/README.md.
 */
 
-
 function sanitize_greyDirection(action) {
-	switch (action) {
-		case "left":
-			return 1;
-		case "right":
-			return 2;
-		case "up":
-			return 3;
-		case "down":
-			return 4;
-
-		default:
-			throw new Error(
-				`GREYAction.GREYDirection must be a 'left'/'right'/'up'/'down', got ${action}`
-			);
-	}
-} 
+  switch (action) {
+    case 'left':
+      return 1;
+    case 'right':
+      return 2;
+    case 'up':
+      return 3;
+    case 'down':
+      return 4;
+
+    default:
+      throw new Error(`GREYAction.GREYDirection must be a 'left'/'right'/'up'/'down', got ${action}`);
+  }
+}
 function sanitize_greyContentEdge(action) {
-	switch (action) {
-		case "left":
-			return 0;
-		case "right":
-			return 1;
-		case "top":
-			return 2;
-		case "bottom":
-			return 3;
-
-		default:
-			throw new Error(
-				`GREYAction.GREYContentEdge must be a 'left'/'right'/'top'/'bottom', got ${action}`
-			);
-	}
-} 
+  switch (action) {
+    case 'left':
+      return 0;
+    case 'right':
+      return 1;
+    case 'top':
+      return 2;
+    case 'bottom':
+      return 3;
+
+    default:
+      throw new Error(`GREYAction.GREYContentEdge must be a 'left'/'right'/'top'/'bottom', got ${action}`);
+  }
+}
 class GREYActions {
   /*@return A GREYAction that performs multiple taps of a specified @c count.
-*/static actionForMultipleTapsWithCount(count) {
-    if (typeof count !== "number") throw new Error("count should be a number, but got " + (count + (" (" + (typeof count + ")"))));
+*/ static actionForMultipleTapsWithCount(count) {
+    if (typeof count !== 'number') throw new Error('count should be a number, but got ' + (count + (' (' + (typeof count + ')'))));
     return {
       target: {
-        type: "Class",
-        value: "GREYActions"
+        type: 'Class',
+        value: 'GREYActions'
       },
-      method: "actionForMultipleTapsWithCount:",
-      args: [{
-        type: "NSInteger",
-        value: count
-      }]
+      method: 'actionForMultipleTapsWithCount:',
+      args: [
+        {
+          type: 'NSInteger',
+          value: count
+        }
+      ]
     };
   }
 
   /*@return A GREYAction that performs multiple taps of a specified @c count at a specified
 @c point.
-*/static actionForMultipleTapsWithCountAtPoint(count, point) {
-    if (typeof count !== "number") throw new Error("count should be a number, but got " + (count + (" (" + (typeof count + ")"))));
-    if (typeof point !== "object") throw new Error("point should be a object, but got " + (point + (" (" + (typeof point + ")"))));
-    if (typeof point.x !== "number") throw new Error("point.x should be a number, but got " + (point.x + (" (" + (typeof point.x + ")"))));
-    if (typeof point.y !== "number") throw new Error("point.y should be a number, but got " + (point.y + (" (" + (typeof point.y + ")"))));
+*/ static actionForMultipleTapsWithCountAtPoint(
```

Comment:
still here!
separate `*/ static actionForMultipleTapsWithCountAtPoint(` to 
```
*/ 
static actionForMultipleTapsWithCountAtPoint(
```

Reply:
Can't do much about it, this would be something we would need to do in prettier itself, providing an upstream PR. Do you really want me to spend the time there? Would be fine with that, just wanted to clarify.

Reply:
It's not related to prettier, it's the code generator